tests/cross_functional/scale/noobaa/test_warp.py

```diff
@@ -129,17 +116,20 @@ def test_s3_benchmark_warp(
         # Check noobaa pods status before running Warp benchmark
         get_noobaa_pods_status()
 
+        # Sleeping script for 1 minute before trigarring warp workload
```

Comment:
Small typo:

```suggestion
        # Sleeping script for 1 minute before triggering warp workload
```

Also, if possible, please add a brief comment to explain why the wait is needed here

Reply:
Fixed